plugins/org.jboss.reddeer.common/src/org/jboss/reddeer/common/condition/WaitCondition.java

```diff
@@ -40,4 +42,28 @@
 	 * @return description of specific wait condition
 	 */
 	String errorMessage();
+	
```

Comment:
After a bit of thought I think that it would probably be safer to just remove errorMessage() method and keep errorMessageWhile/Until. Having a need to implement both methods will at least make a developer think if different message is needed